src/int128_struct_impl.h

```diff
@@ -0,0 +1,129 @@
+#ifndef SECP256K1_INT128_64_IMPL_H
+#define SECP256K1_INT128_64_IMPL_H
+
+#include "int128.h"
+
+#if defined(_M_X64) | defined(_M_ARM64) | defined(_WIN64) /* MSVC */
+  #include <intrin.h>
+  #define secp256k1_umulh __umulh
+  #define secp256k1_mulh __mulh
+#else
```

Comment:
```suggestion
#if defined(_M_X64) | defined(_M_ARM64) | defined(_WIN64) /* MSVC */
  #error
  #include <intrin.h>
  #define secp256k1_umulh __umulh
  #define secp256k1_mulh __mulh
#else
```

Follow-up: I confirmed that change results in an error in the the MSVC build, so the intrinsics should indeed be used.